docs/ceph-fuse-corruption.md

```diff
@@ -0,0 +1,45 @@
+# ceph-fuse: detection of corrupted mounts and their recovery
```

Comment:
It was largely to inform users about the need of node publish secrets, but after discussion with @humblec, and some concrete examples in https://github.com/ceph/ceph-csi/issues/2924 with might not need them anymore, and so this document is probably also not necessary.

Reply:
Also, it describes what the recovery is doing and limits what users can expect from it. I think it still serves this purpose. Please let me know if you'd still like me to remove it.

Reply:
as we can handle this thing internally not sure this is needed and we will [automate the recovery](https://github.com/csi-addons/kubernetes-csi-addons/pull/94) also in the future this will go away, isn't it? am looking will this be staying for a long time or will go away after some time. am fine if all agree to keep this doc.